sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -45,6 +44,13 @@ class WordCountIT(unittest.TestCase):
 
   @attr('IT')
   def test_wordcount_it(self):
+    self._run_wordcount_it()
+
+  @attr('IT', 'ValidatesContainer')
+  def test_wordcount_fnapi_it(self):
+    self._run_wordcount_it(experiment='beam_fn_api')
+
+  def _run_wordcount_it(self, **opts):
```

Comment:
Where is `opts` is used? I am not sure how experiment is passed down.

Reply:
Oh, yes, this was completely lost in the refactor. Thanks for catching this!